### ISSUE TITLE
Import list of source codes and load standard codes

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/Atlas#1674]
### Expected behavior
Under Search we have the import tab. If i add a list of source codes these are added as is. It would be nice to have an option (checkbox?) to translate these on the fly to standard codes that I can add to my concept set.

### Actual behavior
Not implemented now.

### Steps to reproduce behavior
See above.

If there are other easier ways available to do this for a list of source codes also let me know. If not we could make a simple Shiny app for it in the mean time..

If I need to post this new feature request somewhere else let me know.

**Comment by pbr6cornell** [comment]
I agree this would be a very nice feature, it's definitely been a painpoint for me several times (though it's connected to the requirement that you have to be able to specify the source vocabularies of the source  codes, as described in #1810).    I want to make it a requirement for ATLAS 3.0.   @fdefalco , if you can get it into ATLAS 2.8, great, otherwise, we'll push it back to the next version.
